Student_BCOs/(FinalUpdate)Juntian_Wei_BCO_000549.json

```diff
@@ -0,0 +1,434 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000549",
+    "checksum": "b0d6f865943c59544705671ab96555fb4c77a1f39fb8101b80751dc79dc7554c",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "iMir: An integrated pipeline for high-throughput analysis of small non-coding RNA data obtained by smallRNA-Seq",
+        "version": "1.0.0",
+        "license": "https://spdx.org/licenses/CC-BY-4.0.html",
+        "created": "2019-10-16T18:59:23.577045",
+        "modified": "2019-10-17T01:55:13.367413",
+        "review": [
+            {
+                "status": "in-review",
+                "reviewer_comment": "Reviewed by BioCompute technical assistant, approval pending modification by author",
+                "reviewer": {
+                    "orcid": "https://orcid.org/0000-0002-8824-4637",
+                    "affiliation": "George Washington University",
+                    "contribution": [
+                        "curatedBy"
+                    ],
+                    "name": "Janisha Patel",
+                    "email": "janishapatel@gwu.edu"
+                },
+                "date": "10_16_2019"
+            }
+        ],
+        "contributors": [
+            {
+                "name": "Juntian Wei",
+                "orcid": "https://orcid.org/0000-0002-5378-1520",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "jwei48@gwmail.gwu.edu"
+            },
+            {
+                "name": "Giorgio Giurato",
+                "orcid": "https://orcid.org/0000-0002-0538-8978",
+                "affiliation": "Laboratory of Molecular Medicine and Genomics, Department of Medicine and Surgery, University of Salerno, via Allende, 1, Salerno, Baronissi, Italy",
+                "contribution": [
+                    "authoredBy"
+                ],
+                "email": ""
+            },
+            {
+                "name": "Maria Rosaria De Filippo",
+                "orcid": "https://orcid.org/0000-0001-8350-2599",
+                "affiliation": "Fondazione IRCCS SDN, Napoli, Italy",
+                "contribution": [
+                    "authoredBy"
```

Comment:
In addition to "authoredBy", each publication author should also have the "derivedFrom" contribution type. Please add. See #113